src/components/UserMenu/UserMenu.types.ts

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { Network } from '@dcl/schemas/dist/dapps/network'
+import { Avatar } from '@dcl/schemas/dist/platform/profile/avatar'
+import { NotificationsProps } from '../Notifications/Notifications'
+
+export type ManaBalancesProps = {
+  manaBalances?: Partial<Record<Network, number>>
+  onClickBalance?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    network: Network
+  ) => void
+}
+
+export type UserMenuSignedInProps = ManaBalancesProps & {
```

Comment:
As these are props from different components. Would you mind splitting the types and the component files into sub-directories?
For example, we should have a `UserMenuSignedIn` directory with its component and types and a `ManaBalances` as well. This would greatly simplify the understanding of these components which are quite big and have lots of props.

Reply:
I'll move them